frontend/frontend-auto-translate.php

```diff
@@ -145,8 +145,9 @@ public function pre_get_posts( $query ) {
 			$tax = get_taxonomy( $taxonomy );
 			$arr = array();
 			if ( ! empty( $tax ) && ! empty( $qv[ $tax->query_var ] ) ) {
-				$sep = strpos( $qv[ $tax->query_var ], ',' ) !== false ? ',' : '+'; // Two possible separators
-				foreach ( explode( $sep, $qv[ $tax->query_var ] ) as $slug ) {
+				$sep = is_array( $qv[ $tax->query_var ] ) || strpos( $qv[ $tax->query_var ], ',' ) !== false ? ',' : '+'; // Two possible separators
+				$slugarr = is_array( $qv[ $tax->query_var ] ) ? $qv[ $tax->query_var ] : explode( $sep, $qv[ $tax->query_var ] );
+				foreach ( $slugarr as $slug ) {
```

Comment:
What do you think of
```suggestion
				$slugs = array();
				if ( is_array( $qv[ $tax->query_var ] ) ) {
					$slugs = $qv[ $tax->query_var ];
				} elseif ( is_string( $qv[ $tax->query_var ] ) ) {
					$sep   = strpos( $qv[ $tax->query_var ], ',' ) !== false ? ',' : '+'; // Two possible separators.
					$slugs = explode( $sep, $qv[ $tax->query_var ] );
				}
				foreach ( $slugs as $slug ) {
```

Reply:
Looking good 👍